src/components/UserInputComponent/GDOGIDMenuComponent.js

```diff
@@ -36,8 +36,15 @@ export default function GDOGIDMenuComponent(props) {
     setOpen(false);
   };
   const handleTooltipOpen = () => {
-    setOpen(true);
+    setOpen((r) => !r);
```

Comment:
```suggestion
    setOpen((prevValue) => !prevValue);
```
I know I told you to just write `setOpen((r) => !r)`, but it could be changed to be clearer :D